docs/examples/hsv_example.ipynb

```diff
@@ -202,8 +202,17 @@
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
    "version": "3.6.7"
+  },
+  "pycharm": {
```

Comment:
?

Reply:
Guess that's because I did it in pycharm...